ios-xcframework/Config/Gutenberg-Shared.xcconfig

```diff
@@ -34,7 +34,14 @@ DEFINES_MODULE = YES
 DYLIB_COMPATIBILITY_VERSION = 1
 DYLIB_CURRENT_VERSION = 1
 DYLIB_INSTALL_NAME_BASE = @rpath
-ENABLE_MODULE_VERIFIER = YES
+// By default, Xcode creates a project with this setting enabled.
+//
+// We disabled it because otherwise the framework build would fail with
+// "(fatal) cannot find module 'React'".
+//
+// The outcome of this step does not seem to affect the ability to distribute
+// the framework as an XCFramework or how it works when imported in an app.
+ENABLE_MODULE_VERIFIER = NO
```

Comment:
@crazytonyli does this ring any bell for you by any chance?

Reply:
Sorry, I have no idea what this build settings is. A quick Google gives me [this blog](https://mjtsai.com/blog/2023/04/03/xcode-14-3/) and [this Xcode 14.3 release note](https://developer.apple.com/documentation/xcode-release-notes/xcode-14_3-release-notes#New-Features)

> New Features
>
>    Xcode 14.3 includes a new module verifier that generates diagnostics for issues in a framework’s modules. (97345247)